build/task-package.js

```diff
@@ -39,43 +40,115 @@ const IGNORE = [
+                 `tofino Setup ${getManifest().version}${archIfNotx64}.exe`),
+       path.join(Const.PACKAGED_DIST_DIR, `tofino-win32-${ARCH}.exe`)],
+
+      [path.join(Const.PACKAGED_DIST_DIR, `win${archIfNotx64}`,
+                 `tofino-${getManifest().version}-full.nupkg`),
+       path.join(Const.PACKAGED_DIST_DIR, `tofino-${getManifest().version}-full.nupkg`)],
+
+      [path.join(Const.PACKAGED_DIST_DIR, `win${archIfNotx64}`, 'RELEASES'),
+       path.join(Const.PACKAGED_DIST_DIR, 'RELEASES')],
+    ],
+    zips: [
+      [path.join(Const.PACKAGED_DIST_DIR, `win${archIfNotx64}-unpacked`),
+       path.join(Const.PACKAGED_DIST_DIR, `tofino-${PLATFORM}-${ARCH}.zip`)],
+    ],
+  },
+  darwin: {
+    copies: [
+      [path.join(Const.PACKAGED_DIST_DIR, 'mac', `tofino-${getManifest().version}-mac.zip`),
+       path.join(Const.PACKAGED_DIST_DIR, `tofino-${PLATFORM}-${ARCH}.zip`)],
+
+      [path.join(Const.PACKAGED_DIST_DIR, 'mac', `tofino-${getManifest().version}.dmg`),
+       path.join(Const.PACKAGED_DIST_DIR, `tofino-${PLATFORM}-${ARCH}.dmg`)],
+    ],
+  },
+  linux: {
+    zips: [
+      [path.join(Const.PACKAGED_DIST_DIR, 'linux'),
+       path.join(Const.PACKAGED_DIST_DIR, `tofino-${PLATFORM}-${ARCH}.zip`)],
+    ],
+  },
+};
 
 export default async function() {
-  const manifest = BuildUtils.getManifest();
   const electronVersion = getElectronVersion();
   const downloadOptions = getDownloadOptions();
 
   // packager displays a warning if this property is set.
   delete downloadOptions.version;
 
-  const packagedAppPath = await packageApp({
-    arch: ARCH,
-    platform: PLATFORM,
-    ignore: IGNORE,
-    prune: true,
-    version: electronVersion,
-    dir: Const.ROOT,
-    icon: Const.PACKAGED_ICON,
-    out: Const.PACKAGED_DIST_DIR,
-    download: downloadOptions,
+  await builder.build({
+    devMetadata: {
+      directories: {
+        app: Const.ROOT,
+        output: Const.PACKAGED_DIST_DIR,
+      },
+      build: {
+        appId: 'tofino',
+        'app-category-type': 'web',
+        asar: false,
+        icon: Const.PACKAGED_ICON,
+        ignore: IGNORE,
+        electronVersion,
+        download: downloadOptions,
+        npmPrune: true,
+        npmRebuild: false,
+        linux: {
+          target: [],
+        },
+        mac: {
+          icon: path.join(Const.ROOT, 'branding', 'app-icon.icns'),
+          target: ['default'],
+        },
+        dmg: {
+          icon: path.join(Const.ROOT, 'branding', 'app-icon.icns'),
+          contents: [
+            {
+              x: 410,
+              y: 150,
+              type: 'link',
+              path: '/Applications',
+            },
+            {
+              x: 130,
+              y: 150,
+              type: 'file',
+            },
+          ],
+        },
+        win: {
+          icon: path.join(Const.ROOT, 'branding', 'app-icon.ico'),
+          iconURL: 'https://raw.githubusercontent.com/mozilla/tofino/master/branding/app-icon.ico',
```

Comment:
It is used in the windows control panel. I have no idea why a url is needed but there you go.